modules/claims_api/config/schemas/526.json

```diff
@@ -480,6 +480,7 @@
           }
         },
         "confinements": {
+          "description": "Set to true if the Veteran was ever a prisoner of war.",
```

Comment:
Is this correct?  This attribute is a list and not a boolean.

Reply:
@jw81 good catch, I used the other wording that Darcy recommended that doesn't imply a boolean